src/app/modules/home/container-right/container-right.component.ts

```diff
@@ -0,0 +1,23 @@
+import { Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+@Component({
+  selector: 'app-container-right',
+  templateUrl: './container-right.component.html',
+  styleUrls: ['./container-right.component.scss']
+})
+export class ContainerRightComponent {
+
+  constructor(
+      private router: Router
+  ) { }
+
+  public viewAddProject() {
+    if (this.router.url === '/project/add/source') {
```

Comment:
How could the current url be `/project/add` here? And if it was, why would you reload the page?